doc/api/process.markdown

```diff
@@ -59,6 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
+This event is not guaranteed to fire in all situations. For example, if the
+process receives `SIGINT`, it may exit without firing this event.
```

Comment:
I thought the last bullet point might be specific to this: `Note that Windows does not support sending Signals, but Node.js offers some emulation with process.kill(), and child_process.kill():`

So I wasn't sure that it applied in all cases, or if it was specific to signals received via `process.kill()` and `child_process.kill()`. On reflection, that doesn't really make any sense. I wonder if that sentence is actually supposed to have a period at the end rather than a colon.

So yeah, I'll try to edit it a bit more.